docs/process/RepositoryStructure.md

```diff
@@ -0,0 +1,43 @@
+# Repository Structure
+
+This repo contains sample code and definition of Azure DevOps pipelines for CI/CD. These pipelines run in this [Azure DevOps project](https://dev.azure.com/cse-manufacturing/MLOpsManufacturing/_build?view=folders). Azure ML pipelines run in the Azure environment deployed using the Azure DevOps pipelines.
+
+# Folder structure
+
+```bash
+├─ common # contains code common across samples
+│    ├─ infrastructure # code to provision Azure resources to run the samples
+│    └─ pipeline-trigger
```

Comment:
Add an inline comment here?

Reply:
i deleted this as i didn't commit the function app yet